### ISSUE TITLE
Update Armenian translations

### ISSUE THREAD

**Issue by cassmastern** [squidfunk/mkdocs-material#5535]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "hy",
  "action.edit": "Խմբագրել այս էջը",
  "action.skip": "Անցնել պարունակությանը",
  "action.view": "Դիտել այս էջի սկզբնաղբյուրը ⬅️",
  "announce.dismiss": "Այլևս չցուցադրել ⬅️",
  "blog.archive": "Արխիվ ⬅️",
  "blog.categories": "Կատեգորիաներ ⬅️",
  "blog.categories.in": "in ⬅️",
  "blog.continue": "Շարունակել կարդալ ⬅️",
  "blog.draft": "Սևագիր ⬅️",
  "blog.index": "Հետ դեպի ինդեքս ⬅️",
  "blog.meta": "Մետատվյալներ ⬅️",
  "blog.references": "Առնչվող հղումներ ⬅️",
  "clipboard.copy": "Պատճենել",
  "clipboard.copied": "Պատճենված է",
  "consent.accept": "Ընդունել ⬅️",
  "consent.manage": "Կառավարել կարգավորումները ⬅️",
  "consent.reject": "Մերժել ⬅️",
  "footer": "Էջատակ",
  "footer.next": "Հաջորդը",
  "footer.previous": "Նախորդը",
  "header": "Գլխագիր",
  "meta.comments": "Մեկնաբանությունները",
  "meta.source": "Աղբյուր",
  "nav": "Տեղորոշում",
  "readtime.one": "Ընթերցում՝ 1 րոպե ⬅️",
  "readtime.other": "Ընթերցում՝ # րոպե ⬅️",
  "rss.created": "RSS հոսք ⬅️",
  "rss.updated": "Արդիացված բովանդակության RSS հոսք ⬅️",
  "search": "Որոնում",
  "search.placeholder": "Որոնում",
  "search.share": "Կիսվել",
  "search.reset": "Ջնջել",
  "search.result.initializer": "Որոնում",
  "search.result.placeholder": "Մուտքագրեք որոնելու համար",
  "search.result.none": "Արդյունքներ չկան",
  "search.result.one": "1 արդյունք",
  "search.result.other": "# արդյունք",
  "search.result.more.one": "ևս 1-ը այս էջում",
  "search.result.more.other": "ևս #-ը այս էջում",
  "search.result.term.missing": "Բացակայում է",
  "select.language": "Ընտրել լեզուն",
  "select.version": "Ընտրել տարբերակը",
  "source": "Դեպի պահոց",
  "source.file.contributors": "Հեղինակողներ ⬅️",
  "source.file.date.created": "Ստեղծված է",
  "source.file.date.updated": "Վերջին թարմացումը",
  "tabs": "Ներդիրներ",
  "toc": "Բովանդակություն",
  "top": "Վերադառնալ սկիզբ"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 168b8b8fd.

**Comment by squidfunk** [comment]
Released as part of [9.1.14](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.14).
